### DISCUSSION TITLE
Debian Jammy not supported on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Berwynnnn** [telekom-security/tpotce#1172] [Q&A]
I have prepared ISO image by cloning from https://github.com/telekom-security/tpotce and did sudo ./makeiso.sh

However , when running the installer using ./install.sh --type=user, i get an error message saying Aborting. Debian jammy is not supported.

i am currently running Ubuntu 22.04 . Thank you

**Comment by Berwynnnn** [comment]
Solved

**Reply by ihtisamulhasan** [reply]
how to fix this ? i have same problem

**Comment by josmirpa** [comment]
@Berwynnnn  how you fix it? I have the same problem...

**Reply by t3chn0m4g3** [reply]
#1229
